pyproject.toml

```
@@ -58,7 +57,10 @@ dev = [
 scifem = [
   "scikit-fem>=8.1.0" # scikit-fem is a dependency for the multi-dimensional pybamm models
 ]
-all = ["pybop[plot]", "pybop[scifem]"]
+bpx = [
+    "bpx>=0.4",
+]
+all = ["pybop[plot]", "pybop[scifem]", "pybop[bpx]"]
```

This can be simplified to

```suggestion
all = ["pybop[plot,scifem,bpx]"]
```